lib/tran_sock.c

```diff
@@ -651,10 +652,13 @@ tran_sock_fini(vfu_ctx_t *vfu_ctx)
 
     ts = vfu_ctx->tran_data;
 
-    if (ts != NULL && ts->listen_fd != -1) {
-        // FIXME: handle EINTR
-        (void) close(ts->listen_fd);
-        ts->listen_fd = -1;
+    if (ts != NULL) {
+        (void) unlink(vfu_ctx->uuid);
```

Comment:
unlink() can also happen under previous check,
 if (ts != NULL && ts->listen_fd != -1) { }
Not a big gain, but will save one less, useless call to unlink()

Reply:
Right I missed the fact that we dont set  tran_data in case of error in tran_sock_init(). thanks